shared/bls/spectest/aggregate_verify_test.go

```diff
@@ -36,13 +35,18 @@ func testAggregateVerifyYaml(t *testing.T) {
 			require.NoError(t, err)
 			test := &AggregateVerifyTest{}
 			require.NoError(t, yaml.Unmarshal(file, test))
-			pubkeys := make([]iface.PublicKey, 0, len(test.Input.Pubkeys))
+			pubkeys := make([]common.PublicKey, 0, len(test.Input.Pubkeys))
 			msgs := make([][32]byte, 0, len(test.Input.Messages))
 			for _, pubKey := range test.Input.Pubkeys {
 				pkBytes, err := hex.DecodeString(pubKey[2:])
 				require.NoError(t, err)
 				pk, err := bls.PublicKeyFromBytes(pkBytes)
-				require.NoError(t, err)
+				if err != nil {
+					if test.Output == false && err == common.ErrInfinitePubKey {
```

Comment:
```suggestion
					if test.Output == false && errors.Is(err, common.ErrInfinitePubKey) {
```
I think errors.Is is the preferred way to check errors now.